### PR TITLE
KUDO CLI installation update

### PR DESCRIPTION
**Pull request by kensipe**
Fixes: https://github.com/kudobuilder/kudo/issues/1466

Signed-off-by: Ken Sipe <kensipe@gmail.com>

### CODE REVIEW

File: content/docs/cli.md [hunk 1]
```diff
@@ -21,18 +21,20 @@ brew tap kudobuilder/tap
...
 
-Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
+or you can compile and install the plugin from your `$GOPATH/src/github.com/kudobuilder/kudo` root folder via:
 
 ```bash
-kubectl krew install kudo
+make cli-install
 ```
 
+**note:** Building the CLI with `make` will work for all functions with the exception of `kudo init`.  Building and running kudo this way is for developers and advanced users.
```
Comment (porridge) [inline content/docs/cli.md]: I agree it's for advanced users, but in this case we should not be mentioning it here, in the "quick start for newbies" doc.

Putting the warning after the command means that some users will first try it out, and then when it seemingly worked they will read the warning and say "meh".

Also, can you explain why `kudo init` will not work? I think it well might, but the problem is with using unreleased versions of KUDO.

**Review by porridge — CHANGES_REQUESTED** [review]

File: content/docs/cli.md (hunk 1 above)
Comment (kensipe) [reply content/docs/cli.md]: @porridge I think the note should be here as long as we have the `make ...` command as an option here... are you suggesting a move?   I'll take a look if there is a logic place for it.

Comment (porridge) [reply content/docs/cli.md]: I'm suggesting that both the `make ...` option and the note should be removed from this page. Developers and advanced users who want to use the bleeding edge `master` already know how to clone a github repo and read its README.

OTOH, what we seem to be missing here is the simple option to just download the CLI plugin from the asset list of https://github.com/kudobuilder/kudo/releases/latest (latest stable release), `chmod +x` it and put it in the `$PATH`.

Comment (kensipe) [reply content/docs/cli.md]: I completely agree with you on this...   dev details in project repo... this site is for users only!

File: content/docs/cli.md [hunk 2]
```diff
@@ -21,18 +21,26 @@ brew tap kudobuilder/tap
...
 ```
 
-Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
+or you can download the CLI binaries from the release page at https://github.com/kudobuilder/kudo/releases/latest and download the release for your platform and OS.  Make executable and add to your path:
 
 ```bash
-kubectl krew install kudo
+export VERSION=0.12.0
+export OS=$(uname | tr '[:upper:]' '[:lower:]')
+export ARCH=$(uname -m)
```
Comment (porridge) [inline content/docs/cli.md]:
```suggestion
VERSION=0.12.0
OS=$(uname | tr '[:upper:]' '[:lower:]')
ARCH=$(uname -m)
```

File: content/docs/cli.md [hunk 3]
```diff
@@ -21,18 +21,26 @@ brew tap kudobuilder/tap
...
+wget -O kubectl-kudo https://github.com/kudobuilder/kudo/releases/download/v${VERSION}/kubectl-kudo_${VERSION}_${OS}_${ARCH}
+chmod +x kubectl-kudo
+# add to your path
+mv kubectl /usr/local/bin/kubectl-kudo
```
Comment (porridge) [inline content/docs/cli.md]:
```suggestion
sudo mv kubectl-kudo /usr/local/bin/kubectl-kudo
```

Comment (porridge) [reply content/docs/cli.md]: Not sure if `sudo` is a thing on Mac? But the filename needs to change.

File: content/docs/cli.md (hunk 2 above)
Comment (porridge) [reply content/docs/cli.md]: I tend to follow the rule that if an variable does not need to be exported, is should not. Here it's only interpreted by the current shell.

**Review by porridge — APPROVED** [review]

**Pull request merged**